src/app/shared/components/course-card/course-card.component.html

```diff
@@ -1,23 +1,37 @@
 <div class="course-card">
-    <div class="course-card__description">
-      <!-- Replace ::text:: with your code -->
-      <div class="course-card__description-title">::Title::</div>
-      <div class="course-card__description-text">::Description::</div>
+  <div class="course-card__description">
+    <!-- Replace ::text:: with your code -->
+    <div class="course-card__description-title">{{ title }}</div>
+    <div class="course-card__description-text">{{ description }}</div>
+  </div>
+  <div class="course-card__info">
+    <div class="course-card__info-text">
+      <div>
+        <b>Authors: </b>
+        {{ authors.join(", ") }}
+      </div>
+      <div>
+        <b>Duration: </b
+        ><span class="course-card__duration">{{ duration }}</span>
+      </div>
+      <div>
+        <b>Created: </b
+        ><span class="course-card__date">{{ creationDate }}</span>
+      </div>
     </div>
-    <div class="course-card__info">
-        <div class="course-card__info-text">
-            <div>
-              <b>Authors: </b>
-              ::Authors::
-            </div>
-            <div><b>Duration: </b><span class="course-card__duration">::Duration::</span></div>
-            <div><b>Created: </b><span class="course-card__date">::Creation Date::</span></div>
+    <div class="course-card__info-actions">
+      <div (click)="onShowCourse()">
+        <ng-content select="[showCourseBtn]"></ng-content>
```

Comment:
Mentioned in previous comments, using ButtonComponent here directly instead of content projection will increase readability and reduce complexity.